packages/forms/src/Form.js

```diff
@@ -88,7 +88,10 @@ class Form extends React.Component {
 	}
 
 	handleActionClick(onClick) {
```

Comment:
That would not being better to have handleActionClick(onClick=noop) {}
with function noop() {} declared on top ?